schemas/tournamentData.json

```diff
@@ -10,11 +10,34 @@
                     "type": "string"
                 },
                 "source": {
-                    "type": "string"
+                    "type": "string",
+                    "default": "None"
                 },
                 "name": {
                     "type": "string"
-                }
+                },
+				"stages": {
+					"description": "Stage/Bracket ID used by provider",
+					"type": ["array", "null"],
+					"items": {
+						"type": "object",
+                        "properties": {
+                            "name": {
+                                "type": "string",
+                                "description": "Name of stage/bracket"
+                            },
+                            "id": {
+                                "type": "string",
+                                "description": "ID of stage/bracket"
+                            },
+                            "bracketType": {
+                                "type": "string",
+                                "description": "The bracket type, swiss|elimination etc"
```

Comment:
The reason I don't do this is because Battlefy has other we don't support and the idea is if there's other data providers such as SmashGG they may use a different naming scheme.

Reply:
If another service calls a format something else, they should be mapped to be the same on our end to reduce complexity. For instance, if Smash.gg called it "Elim" and Battlefy called it "elimination", they could both become "elimination" when we import a tournament.